src/index.ts

```diff
@@ -230,6 +256,17 @@ class Plugin implements TransformNode {
     return this;
   }
 
+  /**
+   * sets the this.input and this.output on to the the nodes for performing read and write operations
+   * This gets called from broccoli during the TransformNode setup.
+   */
+  setFSFacade(): void {
```

Comment:
Corresponding with my comment over in https://github.com/broccolijs/broccoli/pull/431#discussion_r338141747, I think I'd rather have broccoli pass `input` and `output` in as arguments. I think this will be important when we eventually want to allow the `output` thing to be able to mark the output tree as "dirty".